infrastructure/helm-chart/charts/postgres/values.yaml

```diff
@@ -1 +1,3 @@
+pgUser: postgresadmin
 pgPassword: <pg_password>
+dbName: airy_core
```

Comment:
probably the best name here would be `admin` (as this is the db of the api admin service)